docs/kb/connect-blockchain.md

```diff
@@ -0,0 +1,266 @@
+	blogtypes "github.com/alijnmerchant21/blog/x/blog/types"
+	"github.com/tendermint/starport/starport/pkg/cosmosclient"
+)
+```
+***Tip:** Use your own module instead of `github.com/alijnmerchant21/blog/x/blog/types`*
+
+- **Instance of cosmosclient.**
+
+```go
+func main() {
+
+	cosmos, err := cosmosclient.New(context.Background())
+	if err != nil {
+		log.Fatal(err)	
+	}
+```
+- **Using the account name Alice (obtained while scaffolding the blockchain), define the account address**
+
+```go
+accountName := "alice"
+address, err := cosmos.Address(accountName)
+	if err != nil {
+		log.Fatal(err)
+	}
+```
+
+- **`Create` a new post, `broadcast` it on the chain and `display` the results**
+
+```go
+// Create a post
+msg := &types.MsgCreatePost {
+	Creator: address.String(),
+	Title: "Hello Ali",
+	Body: "This is the first post",
+}
+
+// Broadcast the post
+txResp, err := cosmos.BroadcastTx(accountName, msg)
+if err != nil {
+	log.Fatal(err)
+}
+fmt.Print("MsgCreatePost:\n\n")
+fmt.Println(txResp)
+
+// Display the results
+queryClient := blogtypes.NewQueryClient(cosmos.Context)
+queryResp, err := queryClient.Post(context.Background(), &types.QueryGetPostRequest{})
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Print("\n\nUser:\n\n")
+	fmt.Println(queryResp)
+
+// Tip: To display result of all users, replace POST with POSTALL
+queryResp, err := queryClient.PostAll(context.Background(), &types.QueryAllPostRequest{})
+```
+
+- **Add `go.mod` file**
+```go
+module github.com/tendermint/starport/local_test/client
+
+go 1.17
+
+require (
+	github.com/alijnmerchant21/blog v0.0.0-20210825193134-b6859adfa282
+	github.com/tendermint/starport v0.15.0
+)
+
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/tendermint/starport => github.com/ilgooz/starport v0.0.500
+```
+***Tip:** Instead of `github.com/alijnmerchant21/blog v0.0.0-20210825193134-b6859adfa282` using your own `blog` package*
+
+## Step 3
+To run this program, you need follow these steps:
+- Populate the file `main.go` with the code given above
+- Run `go mod tidy`
+- Run the main file `go run main.go`
+
+**Output**
+```zsh
+MsgCreatePost:
+
+Response:
+  Height: 3522
+  TxHash: E6904A959A4F62B361CAD9C1F8A6976E1B48EB4DA77A1FBB0D41DE921F1E8028
+  Data: 0A100A0A437265617465506F737412020801
+  Raw Log: [{"events":[{"type":"message","attributes":[{"key":"action","value":"CreatePost"}]}]}]
+  Logs: [{"events":[{"type":"message","attributes":[{"key":"action","value":"CreatePost"}]}]}]
+  GasWanted: 300000
+  GasUsed: 45764
+
+
+User:
+
+Post:<creator:"cosmos1u27xu76zamjzgus2py87r8kmafea8cp6rvgtv9" 
+title:"Hello Ali" body:"This is the first post" >
+```
+
+## Step 4
+Similaryly, you can `update` and `delete` the POST
+```go
+// Update a Post
+	msgUpd := &types.MsgUpdatePost{
+		Creator: address.String(),
+		Id:      0,
+		Title:   "Hello Ali",
+		Body:    "This is a modified post",
+	}
+// Delete a Post
```

Comment:
````suggestion
```

## Delete a Post

To delete a post:
````

Reply:
again, I would like to tell the user if they are adding more code to the `main.go` file